### ISSUE TITLE
P.1 recognized as A

### ISSUE THREAD

**Issue by varSEAK** [cov-lineages/pangolin#204]
Hi,
I have updated Pangolin to v.3.0.2. We have some fasta files of the VOCs that we use to check that it still works after an update. B.1.1.7 and B.1.351 worked fine, but when I used our example for P.1 (attached, had to change the file ending to attach it), the result was "A".
I also checked it on the online version (https://pangolin.cog-uk.io/).
I know that lineages can change, but that looks like an error to me?
Thank you very much for investigating/clarifying!

[Brazilian_Variant_P.1.txt](https://github.com/cov-lineages/pangolin/files/6559130/Brazilian_Variant_P.1.txt)

**Comment by AngieHinrichs** [comment]
That is an interesting sequence.  It has no N bases and it covers the whole NC_045512.2 reference sequence but with space characters in locations 11288-11296.  It has none of the mutations that start the B lineage (C241T, C3037T, C14408T, A23403G), which might be why it's called as A now.

Is that a sequence from a physical specimen, or the reference modified to have 15 of the mutations that distinguish P.1 from B.1.1.28 (but none of the mutations that distinguish B.1.1.28 from A)?

**Comment by varSEAK** [comment]
It is indeed a modified reference sequence. When we built it, we did not have a "real" P.1 specimen available.
Up to this version 3, it nevertheless was assigned to P.1. As it should contain at least most of the defining mutations, we were a bit surprised to receive lineage A as a result now.
I have tried again using an actual specimen sequence and it works fine, so I guess I'll just replace that sequence for our update checks.
